docs/general/rendering.md

```diff
@@ -144,6 +147,33 @@ That's the result:
 
 This feature is specific to JsonApi, so you have to use the use the [JsonApi Adapter](adapters.md#jsonapi)
```

Comment:
jsonapi only true for resource-level, also, right?

Might it be helpful to just refer to `json_api/links.md` or `how_to/links.md` for most of the text here to keep the rendering.md slimmer and more general?

ok for followup pr